docs/cryoet_data_portal_docsite_landing.md

```diff
@@ -29,7 +29,7 @@ Here is a summary of the datasets that have been provided. A full description ca
 | Raw movies | Yes | Yes | Yes |
 | Raw tilts | Yes | Yes | No |
 | 3D volumes | Yes | Yes | Yes |
-| Point Annotations (number of particles) | Ribosome, FAS | Ribosome | Ribosome |
+| Point annotations | Ribosome, FAS | Ribosome | Ribosome |
```

Comment:
number of particles is not in the other columns, also use lower case a for consistency in the table